### ISSUE TITLE
Consent Form not display

### ISSUE THREAD

**Issue by MrAdr68** [googleads/googleads-consent-sdk-android#23]
- I selected 9 providers and implement code.
- Add test id .
- setDebugGeography(DebugGeography.DEBUG_GEOGRAPHY_EEA);

But form not displayed:

```
Consent form is not ready to be displayed.
05-24 08:42:57.173 8968-8968/demo.sdk.consent.com.cssample E/demo.sdk.consent.com.cssample: Form loaded
05-24 08:42:57.189 8968-8968/demo.sdk.consent.com.cssample E/demo.sdk.consent.com.cssample: Form loaded
```

**Comment by mikaelvonpfaler** [comment]
@MrAdr68 I ran into the same problem because of the very bad documentation. Try calling the form.show() method only after the form is loaded, in onConsentFormLoaded.

**Comment by rampara** [comment]
@MrAdr68 are you calling `show()` on the `ConsentForm` ? Documentation on the Consent SDK can be found on our [developer docs](https://developers.google.com/admob/android/eu-consent#consent_sdk).

**Comment by MrAdr68** [comment]
@rampara 

Yes, i have already call `.show()` after receiving the `OnConsentFormLoaded()`

`                .withListener(new ConsentFormListener() {
                    @Override
                    public void onConsentFormLoaded() {
                        Log.e(getPackageName(), "Form loaded");
                        form.show();
                    }`

To day, it still display log: Form Error:Consent form is not ready to be displayed.
I tested on Emulator with : `ConsentInformation.getInstance(this).setDebugGeography(DebugGeography.DEBUG_GEOGRAPHY_EEA);`

**Comment by mobipaw** [comment]
@MrAdr68 It is possible that you need to set up your payment info if not already done. I had the same issue and doing so resolved the problem.

**Comment by Izzyjm** [comment]
how did you guys call form.show(); in onConsentFormLoaded ? im getting the error that "form" may have not been initialized yet ?
